caringcaribou/modules/xcp.py

```diff
@@ -468,7 +480,7 @@ def handle_connect_reply(msg):
     with CanActions(arb_id=send_arb_id) as can_wrap:
         print("Attempting XCP memory dump")
         # Connect and prepare for dump
-        can_wrap.send_single_message_with_callback([0xff, 0, 0, 0, 0, 0, 0, 0], handle_connect_reply)
+        can_wrap.send_single_message_with_callback([0xff], handle_connect_reply)
```

Comment:
Why is the padding removed from this specific connect message?

Reply:
Nice question. I think I forgot to put it back as I was testing something.